AirCasting/SearchAndFollow/SearchMap/SearchMapView.swift

```diff
@@ -190,4 +198,22 @@ private extension SearchMapView {
             }
         }
     }
+    
+    var finishButton: some View {
+        Button {
+            isSearchAndFollowFlowActive = false
+            tabSelection.selection = .dashboard
+        } label: {
+            Text(Strings.SearchMapView.finishText)
+                .font(Fonts.muliHeading2.bold())
+                .padding(.trailing, 7)
+        }
+        .overlay(
+            Rectangle()
```

Comment:
There is `Capsule()` which you can use maybe instead and then delete .conrnerRadius? Just idea if not please ignore 😄